### ISSUE TITLE
Loss Function,Gradient Descent and Backpropagation （basic） | Chenkaixuan's Blog

### ISSUE THREAD

**Issue by tju-tomorrow** [tju-tomorrow/tju-tomorrow.github.io#8]
https://tju-tomorrow.github.io/2025/02/23/2025-02-23-Lossfunction/#more 

 更好理解Transformer
